content/contents+de.lr

```diff
@@ -1,76 +1,80 @@
 ---
-year_label_2: Offene **Haushalte**
+year_label_2: Verfolgung der öffentlichen **Geldflüsse**
 ---
-year_label_3: Offene Daten für den **Umweltschutz**
+year_label_3: Offene **Kartographie**
 ---
-year_label_4: Offene Daten für **Menschenrechte**
+year_label_4: Offene Daten für **Gleichstellung**
 ---
 register_text: Veranstaltung hier eintragen
 ---
-events_registered: eingetragene Veranstaltungen
+events_registered: Bisher registrierte Events
 ---
 who_heading: An wen richtet sich der Open Data Day? An jede(n)!
 ---
 who_body:
 
-Wenn du eine Idee zur Verwendung offener Daten hast, egal ob bereits umgesetzt oder noch im Planungsstadium, etwas zur Verwendung offener Daten lernen möchtest oder einfach sehen willst, was am Open Data Day passiert, dann komm einfach bei einem der Events vorbei! Jeder ist herzlich willkommen an den Veranstaltungen teilzunehmen und sich einzubringen. Ganz egal welche Interessen dich mit offenen Daten verbinden oder wie dein aktueller Wissenstand zu offenen Daten ist: Organisatoren von Events rund um den Open Data Day sind bemüht ein interessantes Programm für jede(n) zu bieten!
+Wenn du eine Idee zur Verwendung offener Daten hast, egal ob bereits umgesetzt oder noch im Planungsstadium, etwas zur Verwendung offener Daten lernen möchtest oder einfach bei einem der Open Data Events in Österreich dabei sein möchtest, dann ist der Open Day ganz sicher etwas für dich! Trag ganz einfach deine Idee [hier online](https://docs.google.com/spreadsheets/d/17bnNQshA1gJw0UUa1KBZSCP34NFw4541exaqv2843N0/edit#gid=0) ein, sieh dir die [Liste der bereits organisierten Events](https://fusiontables.google.com/embedviz?viz=CARD&q=select+*+from+1moN6-tpBP5F2gsNkQqryVhoJP3_iZA3qVzAG6LPO&tmplt=3&cpr=2) an oder finde ein [Event auf der Karte](https://fusiontables.google.com/embedviz?q=select+col3+from+1moN6-tpBP5F2gsNkQqryVhoJP3_iZA3qVzAG6LPO&viz=MAP&h=false&lat=47.6924900372039&lng=14.71441445000005&t=1&z=8&l=col3&y=2&tmplt=2&hml=ONE_COL_LAT_LNG). Ganz egal welche Interessen dich mit offenen Daten verbinden oder wie dein aktueller Wissenstand zu offenen Daten ist: Organisatoren von Events rund um den Open Data Day sind bemüht ein interessantes Programm für jede(n) zu bieten!
 ---
 who_title_1: BürgerInnen
 ---
 who_description_1:
 
-Wir brauchen Dich! Offene Daten sind (vor allem) für BürgerInnen, daher gibt es den Open Data Day. Wir brauchen Ideen und Multiplikatoren, um die Idee weiterzuverbreiten!
+Wir brauchen Dich! Offene Daten sind (vor allem) für BürgerInnen, daher gibt es den Open Data Day. Wir brauchen Ideen und Multiplikatoren um die Idee weiterzuverbreiten!
 ---
 who_title_2: MitarbeiterInnen der öffentlichen Verwaltung
 ---
 who_description_2:
 
-Dieser Input ist enorm wichtig! Gemeinsam mit anderen Stakeholdern sind Sie es, die zur Verbesserung der Datenqualität beitragen.
+Dieser Input ist enorm wichtig! Gemeinsam mit anderen Stakeholdern sind sie es, die zur Verbesserung der Datenqualität beitragen.
```

Comment:
The "Sie" was referring to the reader of the website, hence the capitalization. If we want to move away from the formal "Sie" (which we should because we are using the informal "du" in all other parts of the text), I'd suggest:
Gemeinsam mit anderen Stakeholdern seid ihr es, die zur Verbesserung der Datenqualität beitragt.